src/sass/pages/_main-page.scss

```diff
@@ -24,7 +24,7 @@
         grid-gap: 30px;
     }
      @media screen and (min-width: 1024px){
-        grid-template-columns: repeat(3, 1fr);
+        grid-template-columns: repeat(6, 1fr);
```

Comment:
Why is there 6 columns? - There should be 3, according to the mockup.

Reply:
it was necessary to double the number of columns in order to make the last two elements in the center